pkgs/desktops/gnome-3/apps/gnome-todo/default.nix

```diff
@@ -1,28 +1,67 @@
-{ stdenv, fetchurl, meson, ninja, pkgconfig, python3, wrapGAppsHook
-, gettext, gnome3, glib, gtk3, libpeas
-, gnome-online-accounts, gsettings-desktop-schemas
-, evolution-data-server, libxml2, libsoup, libical, librest, json-glib }:
+{ stdenv
```

Comment:
New style choice?

Reply:
Since I am already touching the file, I might as well convert it to a git-friendly format as you suggested.

Reply:
Right, really wish we had formatting rules and tooling to go with it.